spokestack/wakeword/tflite.py

```diff
@@ -59,17 +66,18 @@ def __init__(
         )
         self.encode_width: int = wake_encode_width
         self.state_width: int = wake_encode_width
+        self._hann_window = np.hanning(self.window_size)
 
         self.sample_window: RingBuffer = RingBuffer(shape=[self.window_size])
-        self.fft_window: RingBuffer = RingBuffer(
-            shape=[1, int(self.window_size / 2 + 1)]
-        )
+        self.fft_frame = np.empty((1, int(self.window_size / 2 + 1)), np.float32)
         self.frame_window: RingBuffer = RingBuffer(
             shape=[self.mel_length, self.mel_width]
         )
         self.encode_window: RingBuffer = RingBuffer(
             shape=[self.encode_length, self.encode_width]
         )
+        self.frame_window.fill(np.zeros((1, self.mel_width), np.float32))
```

Comment:
it would be simpler if the buffer could just be filled with a scalar value, instead of having to create a vector

Reply:
yeah I think the numpy `.fill` will be a better way of accomplishing this.